kobo/worker/logger.py

```diff
@@ -66,8 +69,29 @@ def run(self):
                 self._hub.upload_task_log(BytesIO(self._send_data), self._task_id, "stdout.log", append=True)
                 self._send_time = now
                 self._send_data = b""
-            except Exception:
+            except Fault:
                 continue
+            except Exception:
```

Comment:
Thank you, @crungehottman!  I think it would be fine to apply the timeout for all exceptions regardless of their type.

Reply:
Thanks! Fixed